plugin/completion/bin_complete.py

```diff
@@ -151,6 +151,17 @@ def update(self, view, settings):
 
         self.show_errors(view, output_text)
 
+    def get_reference(self, view):
+        """Get location to declaration/definition from given location.
+
+        Args:
+            view (sublime.View): current view
+
+        Returns:
+            Location: location to declaration/definition
+        """
+        sublime.error_message("Not supported for this backend.")
```

Comment:
Probably we would want a more specific message. What did the user do wrong? We should specify that it does not work when "use_libclang" is set to false.